packages/react-day-picker/src/contexts/Focus/FocusContext.tsx

```diff
@@ -64,7 +68,30 @@ export function FocusProvider({
   const { goToMonth, displayMonths } = useNavigation();
   const { numberOfMonths } = useDayPicker();
 
-  const blur = () => setDay(undefined);
+  const modifiersContext = useModifiers();
```

Comment:
Starting to think we should change all these to `useModifiersContext` etc...

Reply:
I think the problem comes from the fact that we ask users to give us `modifiers` (of type Matcher) and then we give users `modifiers` (of type ModifiersStatus) on the way out in `onDayClick`...we use the same name for two different things. But it's not a bad API! It's quite natural to use that word on both sides...

Reply:
I can't think of anything better naming-wise

Reply:
Awesome thanks for the feedback! I've re-thought this countless times  :)